src/components/Profile.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { connect } from 'react-redux';
+
+const Profile = (props) => {
+  const { rockets } = props;
+  const { missions } = props;
+  const reservedRockets = rockets.filter((rocket) => rocket.reserved === true);
+  const reservedMissions = missions.filter((mission) => mission.reserved === true);
+  return (
+    <div className="container-fluid profile">
+      <div className="row align-items-start">
+        <div className="col">
+          <h5>My Missions</h5>
+          {reservedMissions.length === 0 && (
+            <p>There&apos;s no reserved missions</p>
```

Comment:
- Great job handling no rockets/missions found, it was an optional requirement and you added it!